apps/system/lockscreen/js/lockscreen_inputpad.js

```diff
@@ -66,7 +70,7 @@
   LockScreenInputpad.prototype.renderUI = function() {
     return new Promise((resolve, reject) => {
       this.toggleEmergencyButton();
-      this.updatePassCodeUI();
+      this.setPinLength(this.pinLength);
```

Comment:
I prefer to write it as:

    this.setPinLength(this.pinLength);
    this.updatePassCodeUI();

Since it avoid to do too much things inside one function, which is also because the name `setPinLength` looks like a simple setter, not a function drawing thing.